.script/tests/KqlvalidationsTests/YamlFilesTestData/YamlFilesLoader.cs

```diff
@@ -9,15 +11,60 @@ public abstract class YamlFilesLoader
         protected const int TestFolderDepth = 6;
 
         protected abstract List<string> GetDirectoryPaths();
-        
+
         public List<string> GetFilesNames()
         {
-            var directoryPaths = GetDirectoryPaths();
-            return directoryPaths.Aggregate(new List<string>(), (accumulator, directoryPath) =>
+            int prNumber = 0;
+            int.TryParse(System.Environment.GetEnvironmentVariable("PRNUM"), out prNumber);
+            //assign pr number to debug with a pr
+            //prNumber=8414;
+            if (prNumber == 0)
+            {
+                return GetDirectoryPaths()
```

Comment:
Add one log statement that we received PR number as 0. It will help us to debug. Before return.

Reply:
Added the log statement..